src/components/darien/Darien401.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import GenericStatusPage from '../../components/common/GenericStatusPage';
+
+const Darien401 = (props) => {
```

Comment:
I've been switching to writing functional components using declaration rather than expression: ` function MyComponent() {}`. React will correctly use the function name as the component display name this way. For some reason, React won't use it if it's an expression. It really helps with debugging using the React Dev Tools to see the correct names.